mungegithub/github/github.go

```diff
@@ -595,8 +600,41 @@ func (obj *MungeObject) LastModifiedTime() *time.Time {
 	return lastModified
 }
 
-// labelEvent returns the most recent event where the given label was added to an issue
-func (obj *MungeObject) labelEvent(label string) *github.IssueEvent {
+// FirstLabelTime returns the first time the request label was added to an issue.
+// If the label was never added you will get the 0 time.
```

Comment:
will fix.

On Thu, Aug 18, 2016 at 4:12 PM, Eric Paris <notifications@github.com>
wrote:

> In mungegithub/github/github.go
> <https://github.com/kubernetes/contrib/pull/1563#discussion_r75405757>:
>
> > @@ -595,8 +600,41 @@ func (obj *MungeObject) LastModifiedTime() *time.Time {
> >  	return lastModified
> >  }
> >
> > -// labelEvent returns the most recent event where the given label was added to an issue
> > -func (obj *MungeObject) labelEvent(label string) *github.IssueEvent {
> > +// FirstLabelTime returns the first time the request label was added to an issue.
> > +// If the label was never added you will get the 0 time.
>
> If the label was never added you will get nil, not the 0 time (although
> similar)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1563/files/aee73d1142f4fa0657f7e5338b23f4e4ee5f2b3c#r75405757>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAnglqMh3Dk_QqPTR0g1BHGhHj6QqwHyks5qhObzgaJpZM4Jn-Mc>
> .
>